api/specs.proto

```diff
@@ -84,7 +84,7 @@ message ServiceSpec {
 		// Aliases specifies a list of discoverable alternate names for the service on this Target.
 		repeated string aliases = 2;
 	}
-	repeated NetworkAttachmentConfig networks = 7;
+	repeated NetworkAttachmentConfig networks = 7 [deprecated=true];
```

Comment:
How are we handling the deprecated field?

Should it be migrated once and for all or should we have code able to deal with both?